doc/tutorial/machine_learning_map/pyparsing.py

```diff
@@ -32,8 +32,8 @@
 don't need to learn a new syntax for defining grammars or matching expressions - the parsing module
 provides a library of classes that you use to construct the grammar directly in Python.
 
-Here is a program to parse "Hello, World!" (or any greeting of the form 
-C{"<salutation>, <addressee>!"}), built up using L{Word}, L{Literal}, and L{And} elements 
+Here is a program to parse "Hello, World!" (or any greeting of the form
+C{"<salutation>, <addressee>!"}), built up using L{Word}, L{Literal}, and L{And} elements

```

Comment:
A lot of unrelated changes were introduced in this file, please make sure to revert them

Also the whole tutorials part serve mostly as example so I'm not sure it's worth changing them. If there aren'tt a lot of changes, why not

Reply:
Fixed.